bin/packages/get-babel-config.js

```diff
@@ -10,14 +10,7 @@ const babel = require( '@babel/core' );
 const { options: babelDefaultConfig } = babel.loadPartialConfig( {
 	configFile: '@wordpress/babel-preset-default',
 } );
-const plugins = babelDefaultConfig.plugins;
-if ( ! process.env.SKIP_JSX_PRAGMA_TRANSFORM ) {
```

Comment:
what was this environment variable used for? Why it's not useful anymore?

Reply:
oh got it the packages don't require transpilation now

Reply:
yes, I explained it in the description :)